code/api/src/modules/user/types.js

```diff
@@ -42,4 +42,20 @@ const UserGenderType = new GraphQLObjectType({
   })
 })
 
+// //User Results Type
+// const UserResultsType = new GraphQLObjectType({
+//   name: 'userResults',
+//   description: 'User Results Type',
+//
+//   fields:() => ({
+//     id: { type: GraphQLInt },
+//
+//     { tops: { type: GraphQLInt },
+//     bottoms: { type: GraphQLInt },
+//     dresses: { type: GraphQLInt },
+//     accessories: { type: GraphQLInt },
+//     shoes: { type: GraphQLInt }}
+//   })
+// })
+
```

Comment:
During master cleanup, I think we can delete this. This is a remnant of when I was trying to pass a hash to GraphQL